esp/esp/program/forms.py

```diff
@@ -127,7 +127,7 @@ def load_program(self, program):
         pass
 
     def clean_program_modules(self):
-        mods = self.cleaned_data['program_modules'][:] # take a copy of the list to be safe
+        mods = list(self.cleaned_data['program_modules'])[:] # take a copy of the list to be safe
```

Comment:
2 notes:
1. when no boxes are checked and i submit the form and look again, the "will you let students enter a lottery to switch classes" stays checked.
2. The questions are cut off on the right of the page (this appears to not be due to your PR, but a fix would be appreciated in the spirit of the PR)

![image](https://user-images.githubusercontent.com/25392075/119747852-996b8e00-be61-11eb-808c-6c4ec095537d.png)